meterfs/test_meterfs_writeread.c

```diff
@@ -261,10 +261,11 @@ void runner(void)
 
 int main(int argc, char *argv[])
 {
-	file_init(argv[1]);
+	TEST_ASSERT_NOT_NULL(argv[1]);
+	TEST_ASSERT_EQUAL(0, file_init(argv[1]));
 	TEST_ASSERT_EQUAL(0, file_eraseAll());
```

Comment:
I'm not doing full review, just a small note. These asserts under the hood use setjmp/longjmp to jump out of failed tests, so they should only be used in testing and setup/teardown functions.

Reply:
what should be used in this scenario then? classic assert? If followed by return 1?

Reply:
You can print message what kind of error occurred and then return with 1. The runner will fail because there will be not output from unity framework, but it will parse the error message and print it to the user.

Reply:
Ok I'll add message printing